common/production_methods/06_urban_center.txt

```diff
@@ -2033,7 +2033,7 @@ pm_high_rise_offices = {
 	building_modifiers = {
 		workforce_scaled = {
 			building_input_paper_add = 25
-			building_input_computers_add = 5
+			building_input_electronics_add = 5
```

Comment:
"Electronics" is an industrial good representing circuit boards. "Computers" is a consumer good and here is in reference to typewriters.

Reply:
I'll remove this input then, as computers is era later compared to this PM